LZW/LZW/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Data;
+
+namespace LZW
+{
+    class Program
+    {
+        static void Main()
```

Comment:
```suggestion
        static void Main(string[] args)
```
Путь к файлу и ключ должны параметрами к консольному приложению быть.
Их не надо из консоли запрашивать. Их нужно из этого массива брать.